hadoop-hdfs-project/hadoop-hdfs/src/main/java/io/hops/transaction/context/BaseReplicaContext.java

```diff
@@ -231,7 +231,7 @@ private void updateReplicas(INodeCandidatePrimaryKey trg_param,
   }
 
   private void blockDoesNotExist(long blockId, long inodeId){
-    emptyFile(inodeId);
+//    emptyFile(inodeId);
```

Comment:
can you remove the code instead of commenting it out?

Reply:
ok